src/Ast.ml

```diff
@@ -1662,7 +1662,7 @@ end = struct
       | Pexp_setfield (_, _, e0) when e0 == exp -> Some (LessMinus, Non)
       | Pexp_setinstvar _ -> Some (LessMinus, Non)
       | Pexp_field _ -> Some (Dot, Left)
-      | Pexp_send _ -> Some (HashOp, Non)
+      | Pexp_send _ -> Some (Dot, Left)
```

Comment:
You should add a comment to say it's not how it is in the parser and that it is intended.

Reply:
done